policyengine_us/parameters/gov/ed/pell_grant/efc/dependent/asset_modification.yaml

```
@@ -0,0 +1,9 @@
+description: The Education Department counts this share of a dependent student's assets towards the expected family contribution.
+values:
+  2023-01-01: 0.2
+metadata:
+  unit: /1
+  label: Percent of dependent assets for the dependent contribution
```

Borrowing language from the form
```suggestion
  label: EFC dependent asset assessment rate
```